Gemfile

```diff
@@ -49,6 +49,8 @@ gem 'turnout'
 gem 'whenever', require: false
 gem 'yaml_db'
 
+gem 'hashie'
+
```

Comment:
Add gems in alphabetical order in the main block of gems.

Reply:
From the current production bundle:
```
[pub@sul-pub-prod current]$ bundle show --paths | grep hashie
/opt/app/pub/sul-pub/shared/bundle/ruby/2.3.0/gems/hashie-3.4.4
```
Should we add any constraints on the version to bundle?